cypress/integration/pages/articles/testsForCanonicalOnly.js

```diff
@@ -88,6 +88,30 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
             }
           });
         });
+
+        // See https://github.com/bbc/simorgh/pull/4352 for more information.
+        if (service === 'news') {
+          it('plays media when a user clicks play', () => {
+            cy.window().then(win => {
+              const media = getBlockData('video', win.SIMORGH_DATA.pageData);
+              if (media && media.type === 'video') {
+                cy.get(
+                  'div[class^="StyledVideoContainer"] img[class^="StyledImg"]',
+                )
+                  .click()
+                  .should('not.exist')
+                  .then(() => {
+                    cy.get('iframe[class^="StyledIframe"]').then($iframe => {
+                      cy.wrap($iframe.prop('contentWindow'), { timeout: 8000 })
+                        .its('embeddedMedia.playerInstances.mediaPlayer')
+                        .invoke('currentTime')
+                        .should('be.gt', 0);
```

Comment:
Have we tested this test fails if the video loads but does not play? e.g. by suppressing the autoplay option

Reply:
Yes, this fails when presented with a video that cannot be played. E.G.: https://www.test.bbc.co.uk/ws/av-embeds/articles/c3wmq4d1y3wo/p01kdbq2/en-GB